test-utils/runtime-tester/src/fuzzing.rs

```diff
@@ -0,0 +1,174 @@
+                signer_id: signer_account.id.clone(),
+                receiver_id: receiver_account.id.clone(),
+                signer: InMemorySigner::from_seed(
+                    &signer_account.id,
+                    KeyType::ED25519,
+                    &signer_account.id,
+                ),
+                actions: vec![Action::Transfer(TransferAction { deposit: amount })],
+            })
+        });
+
+        // Stake
+        options.push(|u, scope| {
+            let signer_account = scope.get_account(u)?;
+            let amount = u.int_in_range::<u128>(0..=signer_account.balance)?;
+            let signer =
+                InMemorySigner::from_seed(&signer_account.id, KeyType::ED25519, &signer_account.id);
+            let public_key = signer.public_key.clone();
+
+            Ok(TransactionConfig {
+                nonce: scope.get_nonce(),
+                signer_id: signer_account.id.clone(),
+                receiver_id: signer_account.id.clone(),
+                signer,
+                actions: vec![Action::Stake(StakeAction { stake: amount, public_key })],
+            })
+        });
+
+        // Create Account
+        options.push(|u, scope| {
+            let signer_account = scope.get_account(u)?;
+            let new_account = scope.new_account();
+
+            let signer =
+                InMemorySigner::from_seed(&signer_account.id, KeyType::ED25519, &signer_account.id);
+            let new_public_key =
+                InMemorySigner::from_seed(&new_account.id, KeyType::ED25519, &new_account.id)
+                    .public_key;
+            Ok(TransactionConfig {
+                nonce: scope.get_nonce(),
+                signer_id: signer_account.id.clone(),
+                receiver_id: new_account.id.clone(),
+                signer,
+                actions: vec![
+                    Action::CreateAccount(CreateAccountAction {}),
+                    Action::AddKey(AddKeyAction {
+                        public_key: new_public_key,
+                        access_key: AccessKey {
+                            nonce: 0,
+                            permission: AccessKeyPermission::FullAccess,
+                        },
+                    }),
+                    Action::Transfer(TransferAction { deposit: NEAR_BASE }),
+                ],
+            })
+        });
+
+        let f = u.choose(&options)?;
+        f(u, scope)
+    }
+}
+
+#[derive(Clone)]
+pub struct Scope {
+    accounts: Vec<Account>,
+    nonce: Nonce,
+    height: BlockHeight,
+}
+
+#[derive(Clone)]
+pub struct Account {
+    pub id: AccountId,
+    pub balance: Balance,
+}
+
+impl Scope {
+    fn from_seeds(seeds: &Vec<String>) -> Self {
+        let accounts = seeds.iter().map(|id| Account::from_id(id.clone())).collect();
+        Scope { accounts, nonce: 0, height: 0 }
+    }
+
+    pub fn inc_height(&mut self) {
+        self.height += 1
+    }
+
+    pub fn get_height(&self) -> BlockHeight {
```

Comment:
```suggestion
    pub fn height(&self) -> BlockHeight {
```

Rust doesn't use `get_` prefix (and it's ok to have method and field with the same name)